### PR TITLE
Cleanup of PHPDoc return types

### PR DESCRIPTION
**Pull request by RobinMcCorkell**
Pretty much exclusively needing to specify `false` as a valid return type.

### CODE REVIEW

File: lib/public/share_backend.php [hunk 1]
```diff
@@ -46,7 +46,7 @@ public function generateTarget($itemSource, $shareWith, $exclude = null);
 	 * Converts the shared item sources back into the item in the specified format
 	 * @param array $items Shared items
 	 * @param int $format
-	 * @return TODO
```
Comment (LukasReschke) [inline lib/public/share_backend.php]: LOL

Comment (Xenopathic) [inline lib/public/share_backend.php]: Yeah, Scrutinizer was confused by the `\OCP\TODO` return type :P

**Comment by Xenopathic** [comment]
cc @icewind1991 @blizzz

**Comment by DeepDiver1975** [comment]
20 new issues on scrutinizer - hmmm - need to take a closer look. 

Any way: non critical -> 8.1

**Comment by Xenopathic** [comment]
Some of those new issues are caused by Scrutinizer seemingly not rescanning other files that affect these (change the PHPDoc in an interface, but without touching a class implementation, the class implementation still shows the same errors). Perhaps it's possible to rescan everything, after this gets merged? That'll clear up any inconsistencies.

Others are inevitable - the new return types for certain functions means others might have 'unexpected side effects', since they don't handle the possible `false` return. But that's outside the scope of this PR :speak_no_evil:

**Comment by icewind1991** [comment]
:+1:

File: apps/user_ldap/lib/wizard.php [hunk 2]
```diff
@@ -124,7 +124,7 @@ public function countUsers() {
 
 		$usersTotal = $this->countEntries($filter, 'users');
 		$usersTotal = ($usersTotal !== false) ? $usersTotal : 0;
-		$output = self::$l->n('%s user found', '%s users found', $usersTotal, $usersTotal);
+		$output = self::$l->n('%s user found', '%s users found', $usersTotal, array($usersTotal));
```
Comment (PVince81) [inline apps/user_ldap/lib/wizard.php]: Any risk of breakage ? CC @blizzz

Comment (PVince81) [inline apps/user_ldap/lib/wizard.php]: Never mind, it's just a formatting function... /me need :coffee:

**Comment by PVince81** [comment]
Looks fine :+1: 

I wonder how many other PRs will need rebasing after this :wink: 

@DeepDiver1975 you self-assigned, your call to merge this

**Comment by MorrisJobke** [comment]
:+1:

**Pull request merged**